src/home/actions.ts

```diff
@@ -67,3 +78,10 @@ export const startBalanceAutorefresh = () => ({
 export const stopBalanceAutorefresh = () => ({
   type: Actions.STOP_BALANCE_AUTOREFRESH,
 })
+
+export const updateCleverTapInboxMessages = (
```

Comment:
nit: sorry i missed this before too - we've not been so consistent in the past but it's nice to try to model actions as events rather than setters https://redux.js.org/style-guide/#model-actions-as-events-not-setters. maybe something like `cleverTapInboxMessagesReceived`?